### PR TITLE
Update ja.json

### PR DESCRIPTION
**Pull request by Gizel-jiz**
Translation of "tweetIconsPullLeft" .

Also, I don't know what the function of "settingsHeaderGlobalStyling" is, so I can't translate it.
What is this function?
I think it's something to do with headers...

### CODE REVIEW

**Comment by Bl4Cc4t** [comment]
Oh, I'm sorry that it isn't easy to understand. I rearranged the settings a bit and put them into more categories, so it's easier to find them.
`settingsHeaderGlobalStyling` is the hea
image

**Comment by Gizel-jiz** [comment]
Thank you for the detailed information with pictures.
I've just updated it.

However, I couldn't think of a good Japanese translation for "Global Styling", so I couldn't translate it well, and it may have come out a bit unnatural... :(

**Comment by Bl4Cc4t** [comment]
Hm, I think "Global Styling" really sounds a bit weird. What do you think of "Global Look"? I think that sounds better, I think I should change it to that.

**Comment by Gizel-jiz** [comment]
Hmm...
It's true that "Global Look" is easier to convey than "Global Styling", and it's easier to translate into Japanese.
I had a little trouble translating "Styling" into good Japanese when I was translating it. : /
So, I personally think it would be better to change it to "Global Look" : )

**Comment by Gizel-jiz** [comment]
Thank you for modified!
The Japanese translation of "Global Look" has just been updated.  : )

File: i18n/ja.json [hunk 1]
```diff
@@ -31,15 +31,15 @@
 
   "settingsHeaderTimeline": "タイムライン",
   "settingsHeaderSidebars": "サイドバー",
-  "settingsHeaderGlobalLook": "*NEW* Global Look",
+  "settingsHeaderGlobalLook": "*NEW* 全体的な見た目",
```
Comment (Bl4Cc4t) [inline i18n/ja.json:34]:
```suggestion
  "settingsHeaderGlobalLook": "全体的な見た目",
```

File: i18n/ja.json [hunk 2]
```diff
@@ -31,15 +31,15 @@
...
   "settingsHeaderOther": "その他",
   "biggerPreviews": "プレビュー画像の表示を拡張する",
   "biggerPreviewsDesc": "ツイート内のプレビュー画像を完全表示します。",
   "disableAutoRefresh": "自動更新を無効にする（実験的）",
   "disableAutoRefreshDesc": "タイムラインでのツイートのライブ表示を無効にします。<br/>いい結果を期待しないでください。これは常に動作するわけではありません。",
   "expandTcoShortlinks": "t.co リンクを自動的に展開",
   "fontOverride": "*新機能*  カスタムフォントを使用する",
```
Comment (Bl4Cc4t) [inline i18n/ja.json:41]:
```suggestion
  "fontOverride": "カスタムフォントを使用する",
```
Same here.

File: i18n/ja.json [hunk 3]
```diff
@@ -52,14 +52,14 @@
   "leftTrends": "トレンドを左に表示する",
   "leftTrendsDesc": "トレンドを左サイドバーに表示します。",
   "enableQuickBlock": "*新機能* クイックブロックを有効化",
```
Comment (Bl4Cc4t) [inline i18n/ja.json:54]:
```suggestion
  "enableQuickBlock": "クイックブロックを有効化",
```

File: i18n/ja.json [hunk 4]
```diff
@@ -52,14 +52,14 @@
...
-  "enableQuickBlockDesc": "*新機能* フォローボタンの上に3秒間カーソルを置くとブロックすることができます。",
+  "enableQuickBlockDesc": "フォローボタンの上に3秒間カーソルを置くとブロックすることができます。",
   "legacyProfile": "プロフィールページを旧UIのレイアウトに変更する",
   "show10Trends": "常に10個のトレンドを表示する",
   "stickySidebars": "サイドバーの固定",
   "smallSidebars": "小さいサイドバー",
   "smallSidebarsDesc": "350px幅のサイドバーの代わりに290px幅のサイドバーを表示します。",
   "squareAvatars": "四角いアイコン表示",
-  "tweetIconsPullLeft": "*NEW* Pull tweet icons to the left",
+  "tweetIconsPullLeft": "*新機能* ツイートアイコンを左に寄せる",
```
Comment (Bl4Cc4t) [inline i18n/ja.json:62]:
```suggestion
  "tweetIconsPullLeft": "ツイートアイコンを左に寄せる",
```

**Review by Bl4Cc4t — CHANGES_REQUESTED** [review]
The `*NEW*` parts in the strings are only internally used, so that it's more obvious to notice what new parts were added and need to be translated. 
I think I have to add that to the documentation and take a better look at the changes next time...

Related: https://github.com/Bl4Cc4t/GoodTwitter2/pull/108#issuecomment-643733694

**Comment by Gizel-jiz** [comment]
Oh, so that's what happened!
I thought it would be reflected in the settings section, since "NEW" was also translated in the previous translation by someone else.
Before the previous translation (i.e. by me), I had deleted the "NEW" part and submitted a pull request.

Thank you for the correction.
I'll remove the "NEW" as before next time.

**Comment by Bl4Cc4t** [comment]
It looks like you clicked the wrong buttons, the `*NEW*`s are still in the json :D

**Comment by Gizel-jiz** [comment]
Oh, I'm very sorry.
Could you please reflect the correction there?
I'm sorry for the inconvenience.

File: i18n/ja.json (hunk 1 above)
Comment (Bl4Cc4t) [inline i18n/ja.json:34]:
```suggestion
  "settingsHeaderGlobalLook": "全体的な見た目",
```

File: i18n/ja.json (hunk 2 above)
Comment (Bl4Cc4t) [inline i18n/ja.json:41]:
```suggestion
  "fontOverride": "カスタムフォントを使用する",
```

File: i18n/ja.json (hunk 3 above)
Comment (Bl4Cc4t) [inline i18n/ja.json:54]:
```suggestion
  "enableQuickBlock": "クイックブロックを有効化",
```

File: i18n/ja.json (hunk 4 above)
Comment (Bl4Cc4t) [inline i18n/ja.json:62]:
```suggestion
  "tweetIconsPullLeft": "ツイートアイコンを左に寄せる",
```

**Review by Bl4Cc4t — CHANGES_REQUESTED** [review]

**Comment by Bl4Cc4t** [comment]
You need to click on these buttons:
image

**Comment by Gizel-jiz** [comment]
Thank you for the explanatory images. And I apologize for the inconvenience.
I've just authenticated your request. Is there a problem here?

**Comment by Bl4Cc4t** [comment]
Yes! Now you got it!
As always, thanks for your contribution.

**Pull request merged**